### DISCUSSION TITLE
Add issues from other organisations (org project beta)

### DISCUSSION THREAD

**Discussion by janwilmake** [community/community#10709] [Projects and Issues]
It would be great if you would be able to add issues and pr's from other organisations into your organisation-wide projects.
For example, I have many clients that have their own organisation, and more than half of my and my agencies developers work is on those organisations.
Yet, it would still be very nice if we can have all the things in one overview project.
Then you could make it that when you invite people into this org-project, only people that have access to that repo from that other org can see those issues.

**Comment by jason-dour** [comment]
Agree that cross-org functionality would be very welcome! We use numerous organizations, but track projects in one central org.

**Reply by unitof** [reply]
Also quite common for my use case as a freelancer. Some of my clients run their own GitHub org, others prefer to use mine.

The key feature missing here is the ability to invite users or set visibility across multiple organizations, instead of requiring all users to be a member of one.

**Comment by itcarroll** [comment]
~~I'm sure this was intended, but I want to be clear that organizations should include user/personal accounts too. If I'm a collaborator on another user's repository, I would like to be able to add issues from **their** repository to **my** projects.~~

Oh, nevermind. I already can :)

https://github.com/github/feedback/discussions/6212#discussioncomment-1775149

**Reply by ssbarnea** [reply]
While it seems possible to add issue from outside organization to an organization level project, that does not make the project assignment visible to the issue itself.

Is basically only added to the board, and there is no indication on the issue itself, making the feature useless for practical reasons.

**Comment by ayohdev** [comment]
Zenhub has the ability to add issues to a project board from multiple orgs. It would be nice if Github Projects did this by default though.

 https://www.zenhub.com/

**Comment by dmpinder** [comment]
This is possible - https://tosbourn.com/github-projects-different-organisations/

**Reply by rmbaxter67** [reply]
GH docs suggest that this only works for "projects (classic)" rather than the newer versions of projects. My org had no existing "(classic)" projects so I've no way to add notes. The newer "draft issues" don't behave the same way. Shame; this functionality is exactly what I need.

**Comment by johnsonra** [comment]
This doesn't seem to work for a project that I own, though (as opposed to a project owned by a specific org). It would be nice to have a single place to track the issues assigned to me across different orgs. Is there some other option for doing that, which I'm not aware of (other than creating a dummy org for this sole purpose)?

**Comment by schell** [comment]
This would be great for ecosystem projects that are shared between many organizations. At Zcash we have a number of open source orgs looking to collaborate and it would be great to have this automatically in GH projects.

**Comment by jdhenry08** [comment]
Adding my $0.02. I'm in a similar situation as other commentors in that my company manages multiple orgs, but we want a way to centrally manage issues in a single project.

As it stands, this is half-working with very little direction or documentation. I can pull in existing issues from other orgs when in the project view itself, but if I try to create a new issue from the project, even if I link it to the correct org and repo, the project doesn't appear in the issue details. This means I have no way of viewing or editing the project fields I have configured.

All in all, this is a frustrating workflow that feels poorly supported. It could use some love.

**Comment by dkfellows** [comment]
**You can do it**, and I use this a fair bit, but it isn't particularly nice.

* _**Paste the URL for the issue or PR into (one of) the "Add item" box(es) for the project in question.**_ It gives a hint that you can type a PR number or search, but pasting the URL also works and is _easy and fast._
  <img width="153" height="31" alt="image" src="https://github.com/user-attachments/assets/e370a313-c023-4a08-a7e7-2381a19ea6a4" />
  This adds the issue to that project; there's other ways to do it, but they're a lot more long-winded.

The reason it's "not nice" is that you can't see that the issue is in the project from the issue even though you have permission to see all the information required. You can only see the issue from the project, not the other way round. (Of course, you shouldn't be able to see that if the project is private and you're not a member; I'm not describing that case.)
